omniforms/models.py

```diff
@@ -23,6 +23,33 @@
 import re
 
 
+class OmniFieldQuerySet(models.QuerySet):
+    """
+    Custom QuerySet class for the OmniField model
+    """
+    def field_models(self):
```

Comment:
is it worth refactoring this into the function that takes `model_class` as an arg? It looks like this code is repeated in OmniFormHandlerQuerySet.

Not really necessary if these are the only instances though.

Reply:
Done in https://github.com/omni-digital/omni-forms/pull/36/commits/b7166c67da6f2b373634cc19eb0ffd1a9c91fc53